internal/services/signalr/signalr_service_resource.go

```diff
@@ -852,6 +874,12 @@ func resourceArmSignalRServiceSchema() map[string]*pluginsdk.Schema {
 						Required:     true,
 						ValidateFunc: signalrValidate.UrlTemplate,
 					},
+
+					"managed_identity_resource_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.IsUUID,
```

Comment:
why is this a UUID rather than a reference to a Managed Identity like in every other resource?

Reply:
because api also allows system assigned identities, we need to validate the system assigned identity by checking a valid uuid